EnglishFilter/sections/specific.txt

```diff
@@ -244,6 +244,8 @@ relmz.io###relmz-io_728x90
 relmz.io###relmz-io_300x250
 gallons.io###a-left
 gallons.io###a-bottom
+nuggetroyale.io##.mainMenuAdBox
```

Comment:
Move it to Annoyances -> antiadblock

Reply:
Why? Initially this element is ad's leftover.

Reply:
It would be the ad's leftover, if you unblocked `@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=nuggetroyale.io`

Reply:
It's more like:
Ad in the element was blocked > element became leftover > detection occurred > anti-anblocker's message was rendered in leftover

@Alex-302

Reply:
In this case, ok.